test/e2e/user_workload_monitoring_test.go

```diff
@@ -576,6 +573,32 @@ func assertTenancyForMetrics(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			t.Logf("Running query %q", tc.query)
 
+			var cleanupFn func() error
+			// Grant just-enough permissions to the account so it can read metrics.
+			err = framework.Poll(2*time.Second, 10*time.Second, func() error {
+				cleanupFn, err = f.CreateRoleBindingFromTypedRole(userWorkloadTestNs, testAccount, &rbacv1.Role{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "tenancy-test-metrics",
+					},
+					Rules: []rbacv1.PolicyRule{
+						{
+							APIGroups: []string{""},
```

Comment:
The APIGroup should be "metrics.k8s.io".

Reply:
I went with that initially, but it seems that the test fails when that's the case, I've stated my take at why this happens in https://github.com/openshift/cluster-monitoring-operator/pull/2057#issuecomment-1734984657. WDYT?

Reply:
Ah, my bad I thought you meant this for https://github.com/openshift/cluster-monitoring-operator/pull/2057/commits/42172986ae8bace69606e4a05ed47033c9c2aadc#diff-99cf99acf538c21d66511836754b4422d6f67d715ae4f47c6c05ebc78c0b1cbfR739.